src/clr-addons/html-editor/html/html-editor.component.ts

```diff
@@ -1,29 +0,0 @@
-import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
-import { AngularEditorComponent, AngularEditorConfig } from '@kolkov/angular-editor';
-
-@Component({
-  selector: 'clr-html-editor',
-  templateUrl: './html-editor.component.html',
-  host: {
-    '[class.html-editor]': 'true',
-  },
-})
-export class ClrHtmlEditorComponent {
-  @Input()
-  public clrHtmlContent: string;
-  @Input()
-  public clrConfig = {
-    toolbarHiddenButtons: [['insertImage', 'insertVideo', 'fontName']],
```

Comment:
we should mention (at least in the patch notes) that those settings were present before.
I read your comment in the commit about this, just mentioning it here explicitly.

Reply:
will mention it (unless you release the version yourself)